app/javascript/css/application.css

```diff
@@ -18,3 +18,81 @@ td:not(:last-child) {
 th:not(:last-child) {
   border-bottom: 2px solid rgba(0, 0, 0, 0.1);
 }
+
+.btn {
```

Comment:
Para clases custom como estas se puede usar la función [@apply de tailwind](https://tailwindcss.com/docs/functions-and-directives/#apply) para aprovechar sus clases igual